autoload/maktaba/value.vim

```diff
@@ -101,23 +101,37 @@ endfunction
 
 ""
 " Returns the type of {value} as a string.
-" One of "number", "string", "funcref", "list", "dictionary", or "float".
+" One of "number", "string", "funcref", "list", "dictionary", "float",
+" "boolean", "null", "none", "job", or "channel".
 " See also |type()|.
 function! maktaba#value#TypeName(Value) abort
   let l:type = type(a:Value)
   if l:type == 0
-    return "number"
+    return 'number'
   elseif l:type == 1
-    return "string"
+    return 'string'
   elseif l:type == 2
-    return "funcref"
+    return 'funcref'
   elseif l:type == 3
-    return "list"
+    return 'list'
   elseif l:type == 4
-    return "dictionary"
+    return 'dictionary'
   elseif l:type == 5
-    return "float"
+    return 'float'
+  elseif l:type == 6
+    return 'boolean'
+  elseif l:type == 7
+    " v:none exists in Vim, but not Neovim.
```

Comment:
Nit: Clarify "but not Neovim". Looks like it has basically the same type but just calls it "null"?

Reply:
Pushed a fixup, please squash-merge.